src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlCheck.java

```diff
@@ -147,6 +136,34 @@ else if (currentLeaf != null) {
     }
 
     /**
+     * Returns package text.
+     * @param ast PACKAGE_DEF ast node
+     * @return String that represents full package name
+     */
+    public static String getPackageText(DetailAST ast) {
```

Comment:
Why `public`? I prefer to be very strict with modifiers.
Unless there is some reason, which I don't see, this should be `private`.